data/json/items/gun/robofac_gun.json

```diff
@@ -567,10 +596,10 @@
     "material": [ "plastic", "steel" ],
     "symbol": "#",
     "color": "light_gray",
-    "ammo_type": [ "223" ],
+    "ammo_type": [ "223", "300blk" ],
     "flags": [ "MAG_BULKY" ],
     "mag_jam_mult": 2,
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "223": 100 } } ]
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "223": 100, "300blk": 100 } } ]
```

Comment:
I am not confident that 300 BLK works in drums

Reply:
I found listings for combined .300/.223/5.56 drums

Reply:
Can you share them?

Reply:
https://www.highbyoutdoors.com/ProMag-AR-15-Drum-Magazine-223-Rem556-NATO300-Blk--65rd--Black-Polymer_p_77984.html

https://texasshooterssupply.com/product-category/magdrums/magdrums-223_556x45_300aac/

Reply:
The listing for the D-mag is lying, look at questions
![image](https://github.com/user-attachments/assets/6e456b44-95d2-4ae1-9dce-5431f972969d)
https://magpul.com/pmag-d-60-ar-m4-gen-m3.html?mp_global_color=118

For the promag, the actual seller doesn't list it using .300 BLK 
https://promagindustries.com/ar-15-5-56mm-65-rd-black-polymer-drum/